release-please-config.json

```diff
@@ -27,6 +31,9 @@
     }
   ],
   "packages": {
+    ".": {
+      "package-name": ""
```

Comment:
We set this to an empty string since we tag cli releases with only the version and not any name. This enabled release-please to find our old manual releases.